### PR TITLE
i18n zh_CN apidocs page

### PR DESCRIPTION
**Pull request by jiangytcn**
add the missing apidocs page for zh_CN

### CODE REVIEW

File: content/contribute/apidocs.zh.md [hunk 1]
```diff
@@ -18,112 +18,94 @@ aliases: [/contribute/apidocs/]
...
+(https://godoc.org/github.com/jenkins-x/jx)。这两种类型都是由 [jx](https://github.com/jenkins-x/jx) 的代码生成。
 
-## Setup your development environment
 
-It's best to make changes to the Jenkins X code on your local machine. Follow the [development](/development) guide 
-to get set up.
+## 设置你的开发环境
 
-## Writing custom resource documentation
+最好在你的本地电脑上对 Jenkinx X 的代码进行修改。按照[开发](/development)指导文档进行配置。
```
Comment (donhui) [inline content/contribute/apidocs.zh.md]: 链接前后加空格

Comment (jiangytcn) [reply content/contribute/apidocs.zh.md]: thanks @donhui  updated

Comment (jiangytcn) [reply content/contribute/apidocs.zh.md]: @donhui  any issue need to address for this PR?

Comment (donhui) [reply content/contribute/apidocs.zh.md]: > @donhui any issue need to address for this PR?
@jiangytcn 
/lgtm

File: content/contribute/apidocs.zh.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+此外，类型可以通过定义下面的方法来避免引用 "openapi"。下面的例子会和上面的例子产生相同的 OpenAPI 说明：
+
+```go
+    func (_ Time) OpenAPISchemaType() []string { return []string{"string"} }
+    func (_ Time) OpenAPISchemaFormat() string { return "date-time" }
+```
+
+## 编写 Godoc
+
+Jenkins X 使用标准的方法来生成 Godoc，而且会由 [godoc.org](http://godoc.org) 自动生成。这一 [博客](https://blog.golang.org/godoc-documenting-go-code) 为编写 Godoc 提供了很好的介绍。
```
Comment (LinuxSuRen) [inline content/contribute/apidocs.zh.md]: > 这一 [博客]

中文之间不需要有空格

**Pull request merged**